def generate(data):
    msg = "deliberately broken generate function"
    raise Exception(msg)
